### DISCUSSION TITLE
QTree feature but no response

### DISCUSSION THREAD

**Discussion by flowt-au** [quasarframework/quasar#12761] [Ideas / Proposals] [answered]
Hi. I made a small feature request here: [12125](https://github.com/quasarframework/quasar/discussions/12125) a few months ago and since there has been no comment or response I thought I would mention it here in the hope someone could take a look.

Sorry to break protocol, but this little change is important to me and I suspect amongst all you are doing it might have slipped between the cracks.

Thanks,
Murray

**Comment by rstoenescu** [comment, marked as answer]
@flowt-au Will take care of it. Please follow #12762 for the progress.

**Comment by flowt-au** [comment]
Cool! Thanks so much. Much appreciated. :-)
